src/main/java/trade/wayruha/mexc/ws/WebSocketClient.java

```diff
@@ -127,10 +132,14 @@ public void onMessage(WebSocket webSocket, String text) {
         super.onMessage(webSocket, text);
         lastReceivedTime = System.currentTimeMillis();
         log.trace("{} onMessage: {}", logPrefix, text);
-
         try {
-            final T obj = objectMapper.readValue(text, type);
-            callback.onResponse(obj);
+            JavaType typeMap = objectMapper.getTypeFactory().constructParametricType(MexcWSResponse.class, type);
+            MexcWSResponse<T> obj = objectMapper.readValue(text, typeMap);
+            T data = obj.getData();
+            if (nonNull(data) && nonNull(obj.getSymbol())) {
+                data.setSymbol(obj.getSymbol());
+            }
+            callback.onResponse(data);
```

Comment:
According to this code, the data model will always have `symbol`. It may be true with this state of API but eventually they will extend it. For example, they will add 'on balance change subscription' which will have nothing to do with trading symbol (e.g., transfers) and our model will be wrong. I vote against having either 1) base model dto (WSBaseDto) OR a `symbol` as a common field for every dto

Reply:
Ok. Suggest the solution for filling symbol to inner DataModel from wrapper MexcWSResponse.class

Reply:
** they will add ..subscription**
You are very optimistic)

Reply:
so far I have a major concern about this...
I believe we should not include a field `symbol` to the base dto as eventually (and undoubtedly) there will be responses which won't have it.

I can't edit the PR, so could you please update it? let's remove any 'bussiness-related' field (`symbol`) from WSBaseDto.
I can also do it by my own but only after mergin the PR and then ammending (to make history looks good) but it does not worth it...